tamr_unify_client/project/attribute_mapping.py

```diff
@@ -0,0 +1,75 @@
+class AttributeMapping:
+    """see https://docs.tamr.com/reference#retrieve-projects-mappings
+       AttributeMapping and AttributeMappingCollection do not inherit from BaseResource and BaseCollection. BC and BR
+       require a specific URL for each individual attribute mapping (ex: /projects/1/attributeMappings/1),
+       but these types of URLs do not exist for attribute mappings
+       """
+
+    def __init__(self, data):
+        self.data = data
```

Comment:
You may want to make this `self._data = data` to specify that this property should be private